boards/PDM/Src/Io/Io_Efuse.c

```diff
@@ -0,0 +1,373 @@
+#include <assert.h>
+#include <math.h>
+#include <stdlib.h>
+#include "Io_Efuse.h"
+#include "configs/Io_EfuseConfig.h"
+
+#pragma GCC diagnostic ignored "-Wconversion"
+
+struct Efuse_Context
+{
+    // The SPI handle for the SPI device the E-Fuse is connected to
```

Comment:
I think the comments here for the GPIOs aren't necessary since this information is available in the .h file. I'd keep the one for `wdin_bit_to_set` though cuz thats conveying important information

Reply:
removed